src/solve.c

```diff
@@ -86,45 +87,68 @@ void set_run_sequence(void)
 		}
 		move_search_position(step);
 	}
-	run_sequence[i++] = 'M';
+	while (best_neighbor_step(current_walls_around()) == FRONT) {
+		move_search_position(FRONT);
+		run_sequence[i++] = 'F';
+		if (search_distance() != 0)
+			break;
+	}
+	run_sequence[0] = 'B';
+	run_sequence[i++] = 'S';
 	run_sequence[i] = '\0';
 }
 
 /**
  * @brief Execute the speed run movement sequence.
```

Comment:
Include param[in]

Reply:
c0b00f351a5652fa594b0abfc992d3ae99185b28